defaults/main.yml

```diff
@@ -552,7 +548,7 @@ maas_pip_packages:
   - python-keystoneclient
   - python-magnumclient
   - python-neutronclient
-  - python-novaclient<7.0.0
+  - python-novaclient
```

Comment:
Gotcha. Like what is done here? https://github.com/openstack/openstack-ansible-os_nova/blob/stable/ocata/tasks/nova_install.yml#L46-L51